openmmml/models/torchmdnetpotential.py

```diff
@@ -0,0 +1,215 @@
+        name : str
+            The name of the model.
+            'torchmdnet' for a local model file, or pretrained models are available: 'aceff-1.0' or 'aceff-1.1'. 
+        modelPath : str, optional
+            The path to the locally trained torchmdnet model if ``name`` is 'torchmdnet'.
+        lengthScale : float
+            The length conversion factor from the model units to nanometers. 
+            If not specified the default is 0.1 which corresponds to a model in angstrom
+        energyScale : float
+            The energy conversion factor from the model units to kJ/mol.
+            If not specified the default is 96.4916 which corresponds to a model in eV.
+   
+        """
+        self.name = name
+        self.modelPath = modelPath
+        self.lengthScale = lengthScale
+        self.energyScale = energyScale
+
+    def addForces(self,
+                  topology: openmm.app.Topology,
+                  system: openmm.System,
+                  atoms: Optional[Iterable[int]],
+                  forceGroup: int,
+                  **args):
+        # Load the TorchMDNet model.
+
+
+        try:
+            import torchmdnet
+            from torchmdnet.models.model import load_model
+        except ImportError as e:
+            raise ImportError(f"Failed to import torchmdnet please install from https://torchmd-net.readthedocs.io/en/latest/installation.html")
+        import torch
+        import openmmtorch
+
+        includedAtoms = list(topology.atoms())
+        if atoms is not None:
+            includedAtoms = [includedAtoms[i] for i in atoms]
+        numbers = torch.tensor([atom.element.atomic_number for atom in includedAtoms])
+        charge = torch.tensor([args.get('charge', 0)], dtype=torch.float32)
+
+
+        if self.name == 'torchmdnet':
+            # a local path to a torchmdnet checkpoint must be provided 
+            model_file_path = self.modelPath
+        else:
+            from huggingface_hub import hf_hub_download
+
+            if self.name == 'aceff-1.0':
+                repo_id="Acellera/AceFF-1.0"
+                filename="aceff_v1.0.ckpt"
+            elif self.name == 'aceff-1.1':
+                repo_id="Acellera/AceFF-1.1"
+                filename="aceff_v1.1.ckpt"
+            else:
+                raise ValueError(f'Model name {self.name} does not exist.')
+
+
+            model_file_path = hf_hub_download(
+                repo_id=repo_id,
+                filename=filename,
+            )
+
+
+        model = load_model(
+            model_file_path,
+            derivative=False,
+            remove_ref_energy = args.get('remove_ref_energy', True),
+            max_num_neighbors = min(args.get('max_num_neighbors', 64), numbers.shape[0]),
+            static_shapes = True,
+            check_errors = False
+        )
+        for parameter in model.parameters():
+            parameter.requires_grad = False
+
+
+        
+        _batch = args.get('batch', None)
```

Comment:
What is this argument?  You don't mention it either in the API docs or the user guide.

If I understand correctly how batches work in TorchMD-Net, this is mainly relevant during training.  Would there ever be a reason for the user to specify it here?  TorchForce only ever evaluates a single structure at a time.